alca-app/src/components/SubNavigation.jsx

```diff
@@ -1,13 +1,23 @@
-import React from "react";
+import React, { useContext } from "react";
 import { Box, Link } from "@mui/material";
 import { useTheme } from "@emotion/react";
 import { NavLink, useLocation } from "react-router-dom";
 import { PAGES } from "pages/routes";
+import { BalanceContext } from "alice-ui-common";
+import { CountBubble } from "./CountBubble";
 
 export function SubNavigation() {
     const location = useLocation();
     const theme = useTheme();
 
+    const { positions = {} } = useContext(BalanceContext);
+
+    function renderPositionCount(show) {
+        if (!show) return <></>;
+
+        return <CountBubble count={positions.staked.value.length} />;
+    }
```

Comment:
I think we might need to add the lockup count here.
![image](https://user-images.githubusercontent.com/5488554/205155733-45004019-defc-4904-a0ac-94d3065fa9d6.png)
@ACatThatPrograms correct me if I'm wrong.